### PR TITLE
Fix Bootstrap version in docs from 3 to 4

### PR DESCRIPTION
**Pull request by git-no**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | no
| Fixed issues  | none

Fixed links in docs, composer and asset descriptions from Bootstrap 3 to latest Bootstrap 4 version.
Also Bootstrap 4 does not support less anymore, it supports sass.
Also Bootstrap 4 hasn't a theme as Bootstrap 3. Fixed this too.

### CODE REVIEW

**Comment by Thoulah** [comment]
What is that change to swissmation?

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,7 +0,0 @@
-Contributing to Yii2
-====================
-
-- [Report an issue](https://github.com/yiisoft/yii2/blob/master/docs/internals/report-an-issue.md)
```
Comment (samdark) [inline .github/CONTRIBUTING.md]: Why removing it?

File: composer.json [hunk 2]
```diff
@@ -43,12 +28,12 @@
     ],
     "autoload": {
         "psr-4": {
-            "yii\\bootstrap4\\": "src"
+            "swissmation\\bootstrap4\\": "src"
```
Comment (samdark) [inline composer.json]: ?

File: docs/guide-zh-CN/README.md [hunk 3]
```diff
@@ -1,29 +0,0 @@
-基于Yii 2 的 Twitter Bootstrap 扩展
```
Comment (samdark) [inline docs/guide-zh-CN/README.md]: You should not remove translations...

Comment (git-no) [reply docs/guide-zh-CN/README.md]: Sorry, my mistake. I totally agree. Did not know, changes/commits created after creating a pull request are automatically included. Fork is cleaned up now. Pull request should be valid now.

**Comment by samdark** [comment]
Merged. Thanks!

**Pull request merged**